traffic_ops/testing/api/v14/servers_test.go

```diff
@@ -87,6 +87,41 @@ func UpdateTestServers(t *testing.T) {
 		t.Errorf("results do not match actual: %s, expected: %s\n", respServer.Rack, updatedServerRack)
 	}
 
+	// Assign server to DS and then attempt to update to a different type
+	dses, _, err := TOSession.GetDeliveryServices()
+	if err != nil {
+		t.Fatalf("cannot GET DeliveryServices: %v", err)
+	}
+	if len(dses) < 1 {
+		t.Fatal("GET DeliveryServices returned no dses, must have at least 1 to test invalid type server update")
+	}
+
+	serverTypes, _, err := TOSession.GetTypes("server")
+	if err != nil {
+		t.Fatalf("cannot GET Server Types: %v", err)
+	}
+	if len(serverTypes) < 2 {
+		t.Fatal("GET Server Types returned no types, must have at least 2 to test invalid type server update")
```

Comment:
It's possible that it returned 1 type, not no types

Reply:
good catch